### ISSUE TITLE
Notification email subject&header are in server language if from multiple users

### ISSUE THREAD

**Issue by licaon-kter** [mattermost/mattermost-server#10550]
Server: 5.9.0

#### Steps to reproduce
Get message from multiple users.
Get a message from an user.

#### Expected behavior
Get 2 e-mails in my locale

#### Observed behavior (that appears unintentional) 
One e-mail with multiple users messages has subject & header in English (server language) and footer in my locale.

One e-mail from a single user message has subject, header and footer in my locale.

#### Possible fixes
This sounds related to https://mattermost.atlassian.net/browse/MM-2267

**Comment by amyblais** [comment]
Hi @licaon-kter,

Would you be open to sharing more details:

1. Is the language chosen in the System Console for all users or individually from the Account Settings > Language?
2. When you say "multiple users", are you referring to a Group Message channel?
3. What is the language or does this happen with multiple languages?

**Comment by licaon-kter** [comment]
Eg:
1. My locale (in web app): Romanian, default server/client locale (in sys console): English
2. No, I'm away, 2 users chat with me, I get an email with their 2 messages.
3. See above or I don't understand the question

**Comment by amyblais** [comment]
Hi @licaon-kter,

I haven't been able to reproduce this so far - would you be open to sharing any additional relevant information that might help us reproduce the issue, or would you be able to share any screenshots?

**Comment by licaon-kter** [comment]
Green overlay means translated, red means not.
![mmattermail](https://user-images.githubusercontent.com/2282799/55562525-c64a0d80-56fc-11e9-9c81-966d5739beee.PNG)

**Comment by amyblais** [comment]
Hi @licaon-kter,

Thank you for the screenshots, that is helpful.

The main reason I haven't been able to reproduce this is that when I receive email notifications, I always receive them in separate emails instead of combined into one email.

Would you be able to share more details about your environment?

**Comment by licaon-kter** [comment]
Not sure what details.

I have 
```
Enable Email Batching: TRUE

When true, users will have email notifications for multiple direct messages and mentions combined into a single email. Batching will occur at a default interval of 15 minutes, configurable in Account Settings > Notifications.
```
...hence they should be batched, right?

**Comment by amyblais** [comment]
That was it! I'm able to reproduce it now when I enabled email batching. Ticket: https://mattermost.atlassian.net/browse/MM-14930.

Thank you for the report!

**Comment by hanzei** [comment]
The bug fix will be shipped with Mattermost `v5.12`.
